drivers/mmc/mmc.c

```diff
@@ -0,0 +1,680 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Defines a simple and generic interface to access eMMC and SD-card devices.
```

Comment:
Split this into two comments, the regular one used across the codebase and your other line.